source/user-manual/reference/ossec-conf/syscheck.rst

```diff
@@ -44,26 +45,45 @@ Options
 - `max_eps`_
 - `database`_
 
+.. _reference_ossec_syscheck_disabled:
+
+disabled
+^^^^^^^^
+
+Indicates if the syscheck scan is disabled or not.
+
++--------------------+---------+
+| **Default value**  | no      |
++--------------------+---------+
+| **Allowed values** | yes, no |
++--------------------+---------+
+
+Example:
+
+.. code-block:: xml
+
+ <disabled>no</disabled>
+
 .. _reference_ossec_syscheck_directories:
 
 directories
```

Comment:
What is the subtitle style? We need to convert it into blue in order to differentiate them into the list. Please contact with web-team if necessary.

Reply:
I restructured the section. Now, the **syscheck** options are separated with a line.